BuildResidentialHPXML/resources/geometry.rb

```diff
@@ -1427,8 +1442,8 @@ def self.create_doors(runner:,
 
       door_sub_surface = OpenStudio::Model::SubSurface.new(door_polygon, model)
       door_sub_surface.setName("#{min_story_avail_wall.name} - Door")
-      door_sub_surface.setSubSurfaceType('Door')
       door_sub_surface.setSurface(min_story_avail_wall)
+      door_sub_surface.setSubSurfaceType('Door')
```

Comment:
Whoa. Was this breaking anything before? What about in ResStock?

Reply:
I don't know. It was definitely causing an issue in this measure, where the door was being translated to a HPXML Window.

See https://github.com/NREL/OpenStudio/issues/3936.

Reply:
Hmm, then wouldn't there have been no doors in any of the hpxml files created from test osws?